jdaviz/core/loaders/test_loaders.py

```diff
@@ -173,3 +175,21 @@ def test_invoke_from_plugin(specviz_helper, spectrum1d, tmp_path):
     assert len(loader.format.choices) > 0
 
     loader.importer()
+
+
+@pytest.mark.remote_data
+@pytest.mark.parametrize(
+    ('gwcs_to_fits_sip', 'expected_cls'),
+    ((True, WCS),
+     (False, GWCS),))
```

Comment:
```suggestion
@pytest.mark.parametrize(
    ('gwcs_to_fits_sip', 'expected_cls'),
    ((True, WCS),
     (False, GWCS),),
     ids=('True-WCS', 'False-GWCS'))
```

When we run Pytest, it outputs the tests as parametrized by [True-WCS] and [False-WCS] (due to our method of renaming the gwcs WCS import) whereas we really want it to read [True-WCS] and [False-GWCS].  Adding IDs avoids the misconception!